ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.16.yaml

```diff
@@ -22,9 +22,9 @@ build_root:
 images:
 - dockerfile_path: build/Dockerfile.ci
   inputs:
-    openshift_release_rhel-9-release-golang-1.20-openshift-4.15:
+    openshift_release_rhel-9-release-golang-1.20-openshift-4.16:
       as:
-      - registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.20-openshift-4.15
+      - registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.20-openshift-4.16
```

Comment:
Wondering how you verified this image exists/is available? I'd try pulling it manually just to make sure

Reply:
Just pulled it, it does exist